app/services/transaction_service/paypal_events.rb

```diff
@@ -122,11 +121,8 @@ def pending_ext_to_paid(tx)
   end
 
   def delete_transaction(cid:, tx_id:)
-    tx = TransactionModel.where(community_id: cid, id: tx_id, current_state: "initiated").first
-
-    if tx
-      tx.conversation.destroy
```

Comment:
Nothing, they stay as they are so that we if decide to resurrect the transaction we haven't lost them. B conversations with transactions are only ever shown in the context of the transaction so they are filtered out from inbox etc. by looking at the transaction deleted flag.